pogit/codelets/fieldBackground.py

```diff
@@ -10,23 +10,28 @@
 """
             if (cellIdx.y() == ${iy_antenna}  ){
 
-                static constexpr int32_t dim(${dim});
                 static constexpr int32_t pol(${pol});
                 static constexpr float_64 A0_to_J (-4*PI*SI::ELECTRON_MASS_SI/${wavelength} * SI::SPEED_OF_LIGHT_SI / SI::ELECTRON_CHARGE_SI / SI::MUE0_SI / SI::CELL_HEIGHT_SI);
-
+                
                 float_64 distance2_SI = float_64(cellIdx.x()-${ix_cntr}) * SI::CELL_WIDTH_SI * float_64(cellIdx.x()-${ix_cntr}) * SI::CELL_WIDTH_SI ;
-                if (dim==3) {
-                    distance2_SI +=  float_64(cellIdx.z()-${iz_cntr} ) * SI::CELL_DEPTH_SI * float_64(cellIdx.z()-${iz_cntr} ) * SI::CELL_DEPTH_SI;
-                }
-
+                
+                ${distance_3d}
```

Comment:
may be argument name could be more explicit, since it goes to the general `params` dictionary, e.g. `distanceOnAntenna_zComp`